### PR TITLE
Terminal logging

### LINKED ISSUE #46: Logging
It would make sense for JANUS to use the `logging` module, rather than `print()`. This has the benefit of being consistent with PROTEUS, but also is useful for storing model outputs in a more permanent fashion than the terminal history on a given machine.

This should be a relatively easy thing to do.

### LINKED ISSUE #51: Inconsistent SOCRATES environment variables
The new method for accessing the SOCRATES directory defaults to the `SOCRATES` env variable. If this is unset, then it uses the directory provided by `platformdirs`.

However, using `SOCRATES` is inconsistent with other codes, which instead use the variable name `RAD_DIR`. This includes SOCRATES itself, AGNI, and PROTEUS. We should decide which is going to be used going forward.

@stefsmeets what do you think?

### PR DESCRIPTION
**Pull request by nichollsh**
JANUS now logs information to the terminal (stdout, stderr) via the `logging` module, rather than with the `print` function. This integrates better with PROTEUS and offers more flexibility going forward.

This means that PROTEUS will no longer need to redirect stdout to its own logger when JANUS is called. Closes #46.

Replaced "SOCRATES" env variable with "RAD_DIR" for consistency with other modules (although the name is probably less intuitive). Closes #51.

### CODE REVIEW

**Review by stefsmeets — CHANGES_REQUESTED** [review]
Nice effort on cleaning up the print statements!

I have one concern with this PR. 

That is the the non-idiomatic way to define the logger (see [here](https://docs.python.org/3/library/logging.html) for more info). 

```
import logging
logger = logging.getLogger(__name__)
```

This way the logger can be defined by any tool using the library. If JANUS is used as a library, it should not be the one setting up the logger. Instead, the logger should be set up by the CLI or PROTEUS or any other code that calls JANUS.

**Comment by nichollsh** [comment]
Updated this PR to resolve the issues raised above. The loggers are now assigned names in the format of `"fwl."+__name__` as recommended by the Python documentation for libraries.

https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

This means that it's still clear where in the code a logging call is made, but only a single handler is required at a time (or two if you are also logging to a file). There will be a corresponding update to PROTEUS.

Some other minor changes including refactoring of some functions and documentation.

File: src/janus/utils/logs.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Simple terminal-logger instance 
+def setup_logger(level:str="INFO"):
+
+    custom_logger = logging.getLogger("fwl."+__name__)
+    custom_logger.handlers.clear()
+
+    level = str(level).strip().upper()
+    if level not in ["INFO", "DEBUG", "ERROR", "WARNING"]:
+        raise Exception("Invalid log level '%s'"%level)
```
Comment (stefsmeets) [inline src/janus/utils/logs.py:12]:
```suggestion
        raise ValueError("Invalid log level '%s'"%level)
```

**Review by stefsmeets — APPROVED** [review]
Looks great! Definitely a nice step forward, and clever idea by prefixing `fwl.` to the logger name. This seems in line with documentation on how it was intended to be used, curious to see how it will work in practice 😅

**Pull request merged**